packages/manager/apps/web/client/app/dns-zone/new/dns-zone-new.controller.js

```diff
@@ -27,7 +27,7 @@ export default class newDnsZoneCtrl {
       `${this.user.ovhSubsidiary}.guides.dnsForExternalDomain`,
     );
     this.configuration = {
-      zone: null,
+      zone: '',
```

Comment:
null as default value makes more sense than an empty string. Any reason you changed it?

Reply:
Thanks @radireddy for point out.

The diff came from a previous commit which is not related to the scope of this Pull Request (it has been drop via a rebase).

The branch `feature/MANAGER-5488` has been successfully rebased on `develop`.